Sources/SFSafeSymbols/Symbols/SFSymbol+3.1.swift

```diff
@@ -4,41 +4,90 @@
 @available(iOS 15.1, macOS 12.0, tvOS 15.1, watchOS 8.1, *)
 public extension SFSymbol {
     /// 􁉘
+    /// Single Localization, 2 Layersets
+    ///
+    /// Layersets:
+    /// - Monochrome
```

Comment:
Unlike localizations, Like the app, I think it's better to always mention `Monochrome` even for symbols with a single layerset.